qt/widgets/common/inc/MantidQtWidgets/Common/PropertyHandler.h

```diff
@@ -205,6 +205,7 @@ class EXPORT_OPT_MANTIDQT_COMMON PropertyHandler : public QObject, public Mantid
   QString updateStructureTooltip();
 
   QMap<QString, QtProperty *> getTies() { return m_ties; };
+  bool hasTies() { return !m_ties.isEmpty(); }; // added for testing purposes
```

Comment:
I would drop the comment, has ties is potentially useful beyond testing. 
For example developers, can check if something hasTies and act accordingly whereas this comment might steer some people away or become stale once someone uses it outside testing